Documentation/devicetree/bindings/hwmon/microchip,emc2305.yaml

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
+%YAML 1.2
+---
+
+$id: http://devicetree.org/schemas/hwmon/emc2305.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Microchip EMC2305 RPM-based PWM Fan Speed Controller
+
+properties:
+  compatible:
+    enum:
+      - microcip,emc2305
```

Comment:
1. Typo.
2. The driver uses `smsc,emc2305`.

Reply:
Bindings are from the patch submitted upstream which Rob H has already slated. I guess we ought to fix them up if we're applying it downstream.

https://www.microchip.com/investor/Pressrelease/MCHP%20and%20SMSC%20Announce%20the%20Acquisition%20of%20SMSC%20by%20MCHP.050212.pdf
So SMSC is now Microchip.
I guess we add both to binding and driver.

Reply:
Since this is a new driver, I see no reason not to use `microchip` everywhere.

Reply:
OK, done.